pkg/virt-config/feature-gates.go

```diff
@@ -29,7 +29,7 @@ const (
 	NUMAFeatureGate   = "NUMA"
 	IgnitionGate      = "ExperimentalIgnitionSupport"
 	LiveMigrationGate = "LiveMigration"
-	// SRIOVLiveMigrationGate enable's Live Migration for VM's with SRIOV interfaces.
+	// SRIOVLiveMigrationGate enables Live Migration for VM's with network SR-IOV interfaces.
 	SRIOVLiveMigrationGate     = "SRIOVLiveMigration"
```

Comment:
Why isn't it removed?

Reply:
According to https://github.com/kubevirt/kubevirt/blob/main/docs/deprecation.md: 
```
When the feature gate is enabled (by creating / updating Kubevit CR), a warning should be presented to the user. The best fit to trigger the warning is probably our validating webhooks.
```
It was left in order to be used when the warning is raised.

Reply:
So where is the warning?:)

Reply:
When you add the feature gate to kubevirt CR you get the following warning:
```
Warning: feature gate SRIOVLiveMigration is deprecated, therefore it can be safely removed and is redundant. For more info, please look at: https://github.com/kubevirt/kubevirt/blob/main/docs/deprecation.md
```

Reply:
@AlonaKaplan [This](https://github.com/kubevirt/kubevirt/pull/8047/commits/2e5d20e248fdb029ea6bbc25989d035d4b73bb1a) triggers the warning.